packages/energy-api-influxdb/test/reads.controller.ts

```diff
@@ -38,6 +39,15 @@ export class ReadsController extends BaseReadsController {
     return super.getReadsAggregates(meterId, filter);
   }
 
+  @Get('/:meter/latest')
+  public async getLastRead(
```

Comment:
Please rename the method to `getLatestRead` to follow the rename on the API level.